### ISSUE TITLE
Release Spark provided fix to maven

### ISSUE THREAD

**Issue by dszakallas** [banzaicloud/spark-metrics#62]
**Describe the bug**

`spark-core` is a compile dependency for the artifact in maven central, causing merge issues when building an assembly.
```xml
<dependency>
  <groupId>org.apache.spark</groupId>
  <artifactId>spark-core_2.11</artifactId>
  <version>2.4.4</version>
</dependency>
```
https://repo1.maven.org/maven2/com/banzaicloud/spark-metrics_2.11/2.4-1.0.5/spark-metrics_2.11-2.4-1.0.5.pom

**Expected behavior**

`spark-core` should be a provided dependency.
As a workaround I had to add

```scala
"com.banzaicloud" %% "spark-metrics"  % "2.4-1.0.5" excludeAll(ExclusionRule(organization = "org.apache.spark"))
```
to my dependencies.

**Additional context**

Fix has already been applied, but not released. 
https://github.com/banzaicloud/spark-metrics/commit/9c5a5178f0d877911aba93ff832c4c1e559c83da

**Comment by stoader** [comment]
Fix released as 2.4-1.0.6

**Comment by dszakallas** [comment]
Thanks!
